### ISSUE TITLE
Retain aspect ratio

### ISSUE THREAD

**Issue by kryptus36** [nativescript-community/ui-lottie#91]
If I set the width of my lottie and not height, it will shrink the animation itself, but the container retains the original height of the lottie file. I'm trying to overlay an animated check mark over the bottom right corner of another image and it doesn't seem possible.

```xml
<FlexboxLayout id='check-in-buttons'>
          <StackLayout>
            <AbsoluteLayout>
              <Image src="~/assets/images/buttons/check-in-facebook.png" tap="{{ shareFacebook }}" />
              <Lottie:LottieView src="~/assets/lottie/check.json" loop="true" autoPlay="true" visibility="{{ checkIn.facebook ? 'visible' : 'collapsed' }}" />
            </AbsoluteLayout>
            <Label text="100 points" />
          </StackLayout>
          <StackLayout>
            <AbsoluteLayout>
              <Image src="~/assets/images/buttons/check-in-twitter.png" />
              <Lottie:LottieView src="~/assets/lottie/check.json" loop="true" autoPlay="true" visibility="{{ checkIn.twitter? 'visible' : 'collapsed' }}" />
            </AbsoluteLayout>
            <Label text="100 points" />
          </StackLayout>
        </FlexboxLayout>
```

```scss
#check-in-buttons {
  width: 100%;
  align-items: flex-start;
  justify-content: center;
  stacklayout {
    width: 20%;
    margin: 7;

    lottieview {
      width: 15%;
      top: 75%;
      left: 75%;
    }

    image {
      width: 100%;
    }
  }
  label {
    text-align: center;
  }
}
```

My lottie animation ends up the size I expect and placed correctly on the x axis. But the bounding box around the animation is way tall and the animation is way lower than it should be.

I think it should act more like an image, if I set width and not height, height is determined by the aspect ratio. It seems as though this is true for the animation, but not its container.

**Comment by farfromrefug** [comment]
@kryptus36 what platform?

**Comment by farfromrefug** [comment]
@kryptus36 should be fixed in 4.3.2 for iOS/ Android. Also added the `stretch` property

**Comment by kryptus36** [comment]
Much better! Thanks!
